com.haysell.pos.json

```diff
@@ -0,0 +1,39 @@
+{
+    "id":"com.haysell.pos",
+    "runtime":"org.kde.Platform",
+    "runtime-version":"5.15-23.08",
+    "sdk":"org.kde.Sdk",
+    "command":"haysell-pos",
+    "finish-args":[
+      "--share=ipc",
+      "--share=network",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--socket=pulseaudio",
+      "--filesystem=host"
+    ],
+    "cleanup":[
+      "/include",
+      "*.la",
+      "*.a"
+    ],
+    "modules":[
+      {
+        "name":"haysell-pos",
+        "buildsystem":"cmake-ninja",
+        "sources":[
+          {
+            "type":"git",
+            "url":"https://github.com/Delphi251189/haysell-pos.git",
+            "commit" : "fd43a51911927a15e118b63a7f4d5fadb363baa8"
+          }
+        ],
+        "post-install": [
+            "install -Dm0644 flatpak/com.haysell.pos.metainfo.xml /app/share/metainfo/com.haysell.pos.metainfo.xml",
+            "install -Dm0644 flatpak/com.haysell.pos.desktop /app/share/applications/com.haysell.pos.desktop",
+            "install -Dm0644 libs/haysell/Icons/logo/256x256.png /app/share/icons/hicolor/256x256/apps/com.haysell.pos.png"
```

Comment:
given that none of this is flatpak specific, ideally a future release should install this as part of the build process, like most other do. Not a blocker at the moment.

Reply:
> given that none of this is flatpak specific, ideally a future release should install this as part of the build process, like most other do. Not a blocker at the moment.

Yes, the project structure must be changed soon and some common files ( like logos, icons and some other medias)should move out of libs/haysell directory 

Can I merge at this moment?